pandas/core/dtypes/concat.py

```diff
@@ -183,6 +185,15 @@ def concat_categorical(to_concat, axis: int = 0):
     return result
 
 
+def _safe_array(x):
```

Comment:
so i actually like this as a real function; can you move to pandas.compat.numpy and use everywhere?

Reply:
can you do this one